### PR TITLE
docs: improve how we start bytebase in docker and connect a local mysql database

### PR DESCRIPTION
**Pull request by tianzhou**
* Move running local MySQL to the install section
* Improve add an instance doc and screenshot
* Clarify --add-host host.docker.internal:host-gateway --network host usage
* Update corresponding blog post link

### CODE REVIEW

File: content/docs/en/get-started/configure-workspace/add-an-instance.md [hunk 1]
```diff
@@ -7,22 +7,28 @@ title: Add an Instance
...
 6. Choose one instance, scroll down, here you can see all the databases in that instance. If you unchecked **Sync schema now** in the previous step, now you can click **Sync now** to wait for all databases to appear.
 
-![create-an-instance](/docs/en/get-started/configure-workspace/add-an-instance/create-an-instance.webp)
+![create-an-instance](/docs/en/get-started/configure-workspace/add-an-instance/add-an-instance.webp)
 
 <hint-block type="info">
 
-Environment can not be changed once the Instance has been created.
+1. Environment can not be changed once the Instance has been created.
+2. The screeshot assumes you run Bytebase inside Docker and try to connect to a database instance on the same host. Thus it uses `host.docker.internal`. If Bytebase is not running inside Docker, you should supply the normal `127.0.0.1` to connect the instance on the same host [(detailed explanation)](https://stackoverflow.com/a/24326540/235983).
```
Comment (RainbowDashy) [inline content/docs/en/get-started/configure-workspace/add-an-instance.md:30]:
```suggestion
2. The screenshot assumes you run Bytebase inside Docker and try to connect to a database instance on the same host. Thus it uses `host.docker.internal`. If Bytebase is not running inside Docker, you should supply the normal `127.0.0.1` to connect the instance on the same host [(detailed explanation)](https://stackoverflow.com/a/24326540/235983).
```

**Review by h3n4l — APPROVED** [review]
LGTM

File: content/docs/en/get-started/configure-workspace/add-an-instance.md [hunk 2]
```diff
@@ -7,22 +7,28 @@ title: Add an Instance
...
 6. Choose one instance, scroll down, here you can see all the databases in that instance. If you unchecked **Sync schema now** in the previous step, now you can click **Sync now** to wait for all databases to appear.
 
-![create-an-instance](/docs/en/get-started/configure-workspace/add-an-instance/create-an-instance.webp)
+![create-an-instance](/docs/en/get-started/configure-workspace/add-an-instance/add-an-instance.webp)
 
 <hint-block type="info">
 
-Environment can not be changed once the Instance has been created.
+1. Environment can not be changed once the Instance has been created.
+2. The screenshot assumes you run Bytebase inside Docker and try to connect to a database instance on the same host. Thus it uses `host.docker.internal`. If Bytebase is not running inside Docker, you should supply the normal `127.0.0.1` to connect the instance on the same host [(detailed explanation)](https://stackoverflow.com/a/24326540/235983).
```
Comment (adela-bytebase) [inline content/docs/en/get-started/configure-workspace/add-an-instance.md:30]: For the previous version, it seems we should always use `host.docker.internal`, I will try the new version.

**Review by adela-bytebase — APPROVED** [review]
LGTM

**Pull request merged**